### DISCUSSION TITLE
How to display the image only on the second monitor?

### DISCUSSION THREAD

**Discussion by durko74** [ltsp/ltsp#271] [Q&A]
I want to use a laptop without my own screen, but an external monitor is connected to it; how can I display an image on it?

**Comment by alkisg** [comment]
Hi durko74, this question isn't specific to LTSP so you should first direct it to your distribution community.

The answer of the community might be "create a xorg.conf" or "use the control panel > monitor settings" or "run an xrandr command from /etc/xdg/autostart" or various other methods.

If you decide to use a xorg.conf, and if the distribution community helped you to find the correct contents, you may then read the [examples section of the ltsp.conf man page](https://ltsp.org/man/ltsp.conf/#examples) to see how to send a custom xorg.conf to a specific client only.
